### ISSUE TITLE
Tracking

### ISSUE THREAD

**Issue by grayjedismurf** [kylewill0725/EDPirateMissionTracker#12]
![Untitled](https://user-images.githubusercontent.com/84148370/118164047-82e02400-b3d7-11eb-9a76-bebfe7e7d412.png)
I had several missions expire while I was gone for an extended period and now the tracker is tracking several missions as negatives and they will not go away even when I have zero missions in my log. I have uninstalled and reinstalled several times. I am unsure of where the app pulls info from to clear out any cache that might be left over.

**Comment by kylewill0725** [comment]
1) Close the game
1) Right click the Windows start button
1) Click "Run"
1) Copy  
`%userprofile%\Saved Games\Frontier Developments\Elite Dangerous`  
and paste it in the text box
1) Click "OK"
1) Create a new folder called "Old Journals"
1) Move every file that starts with "Journal" into the "Old Journals" folder
    - (You may want these journals in the future for things like EDDiscovery or EDSM)
1) Restart the game  
  
It should work now.

**Comment by grayjedismurf** [comment]
Just to make sure I don't mess anything up. Should I wait until I finish
the stack I just built before doing this?

**Comment by kylewill0725** [comment]
Yes. This completely resets the data that the program works with.

**Comment by grayjedismurf** [comment]
Thanks!
